docs/magento2/search-engine.mdx

```diff
@@ -405,8 +414,94 @@ Then, you need to define the API url in your environment variables:
 FRONT_COMMERCE_ATTRAQT_RECOMMENDATION_API_URL=http://api.early-birds.io/widget
 ```
 
-After this step, you should be able to use this loader in your modules to fetch
-Attraqt recommendations for your widgets.
+#### Using Attraqt Orchestrator Chrome extension
+
+In order to use this Attraqt Orchestrator extension for Google Chrome,
+Front-Commerce exposes a GraphQL interface and provides a front-end component
+which does the needed setup. In this example, we'll make a module
+`SuggestedProduct` that fetches a recommendation from Attraqt via
+Front-Commerce's loader, and use it to initialize Attraqt Orchestrator
+
+:::note
+
+See [Extend the GraphQL schema](/docs/essentials/extend-the-graphql-schema) for
+instructions on how to add your own GraphQL module.
+
+:::
+
+You will first need to update your GraphQL schema to implement this interface
+and use it in a query:
+
+```graphql title="my-module/server/modules/SuggestedProducts/schema.gql"
+type AttraqtRecommendation implements AttraqtRecommendationResult {
+  id: ID
+}
+
+extend type Query {
+  attraqtRecommendation(attraqtWidgetId: String!): AttraqtRecommendation
```

Comment:
For the sake of the exemple, this query only uses the widget ID, however in a more real usecase the `context` should be passed as well.

Reply:
I'd suggest to even remove the `attraqtWidgetId` and consider it has a hardcoded value called in the resolver (with context if needed), as an example.

```suggestion
  productRecommendation: ProductRecommendation
```